Annotation/constituent_golden_set.json

```diff
@@ -453,7 +520,7 @@
                     }
                 ],
                 "type": "constituent",
-                "clause": "",
+                "clause": "how much money has been wasted on items the child does not play with",

```

Comment:
Again, degree so maybe not interrogative. It actually seems to me more like a free relative?

Reply:
I'll remove this one